### PR TITLE
Add maintain command and other small updates

### PR DESCRIPTION
**Pull request by jiajunteo**

### CODE REVIEW

File: src/main/java/seedu/thanepark/logic/commands/ShutDownCommand.java [hunk 1]
```diff
@@ -81,7 +81,7 @@ private static Ride createUpdatedRide(Ride rideToOpen, UpdateRideDescriptor open
         Name updatedName = openRideDescriptor.getName().orElse(rideToOpen.getName());
         Maintenance updatedMaintenance =
                 openRideDescriptor.getMaintenance().orElse(rideToOpen.getDaysSinceMaintenance());
-        WaitTime updatedWaitTime = openRideDescriptor.getWaitTime().orElse(rideToOpen.getWaitingTime());
+        WaitTime updatedWaitTime = new WaitTime(0);
```
Comment (YuPeiHenry) [inline src/main/java/seedu/thanepark/logic/commands/ShutDownCommand.java]: Good catch.

File: src/main/java/seedu/thanepark/logic/parser/ShutDownCommandParser.java [hunk 2]
```diff
@@ -8,16 +8,14 @@
 import seedu.thanepark.logic.parser.exceptions.ParseException;
 
 /**
- * Parses input arguments and creates a new OpenCommand object
+ * Parses input arguments and creates a new ShutDownCommand object
```
Comment (YuPeiHenry) [inline src/main/java/seedu/thanepark/logic/parser/ShutDownCommandParser.java]: Good catch.

File: src/main/java/seedu/thanepark/model/ride/Ride.java [hunk 3]
```diff
@@ -68,6 +68,10 @@ public Maintenance getDaysSinceMaintenance() {
         return daysSinceMaintenance;
     }
 
+    public Maintenance resetMaintenance() {
+        return new Maintenance("0");
```
Comment (YuPeiHenry) [inline src/main/java/seedu/thanepark/model/ride/Ride.java]: Is this supposed to be `daysSinceMaintenance = new Maintenance("0");`
Also, might want to make `Maintenance` immutable. Currently we can `setValue()`.

File: src/main/java/seedu/thanepark/ui/RideCard.java [hunk 4]
```diff
@@ -54,12 +54,13 @@ public RideCard(Ride ride, int displayedIndex) throws AccessibilityException {
         statusString.setText(ride.getStatus().name());
         if (ride.getStatus().equals(Status.OPEN)) {
             statusString.setStyle("-fx-background-color: #27D858");
-        } else if (ride.getStatus().equals(Status.SHUTDOWN)
-            || ride.getStatus().equals(Status.MAINTENANCE)) {
+        } else if (ride.getStatus().equals(Status.SHUTDOWN)) {
             statusString.setStyle("-fx-background-color: #CC3045");
-        //error
+        } else if (ride.getStatus().equals(Status.MAINTENANCE)) {
+            statusString.setStyle("-fx-background-color: #cc6a26");
```
Comment (YuPeiHenry) [inline src/main/java/seedu/thanepark/ui/RideCard.java]: Can we standardize? Lower case or upper case hex will do.

**Review by YuPeiHenry — CHANGES_REQUESTED** [review]
Good job overall, thanks for adding maintain command. Please look at the concerns listed, address them and I will approve.

File: src/main/java/seedu/thanepark/ui/RideCard.java (hunk 4 above)
Comment (jiajunteo) [reply src/main/java/seedu/thanepark/ui/RideCard.java]: Ok i'll keep it to uppercase hex

**Review by YuPeiHenry — APPROVED** [review]

**Pull request merged**